build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -9,7 +9,7 @@
   <UsingTask TaskName="DotNetTest" AssemblyFile="$(CLIBuildDll)" />
   <UsingTask TaskName="DownloadFile" AssemblyFile="$(CLIBuildDll)" />
   <UsingTask TaskName="ExtractArchive" AssemblyFile="$(CLIBuildDll)" />
-  <UsingTask TaskName="ArchiveDirectory" AssemblyFile="$(CLIBuildDll)" />
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(BuildToolsDir)/Microsoft.DotNet.Build.Tasks.dll"/>
```

Comment:
This is a very weird name for a task. But I understand it is coming from BuildTools.